kadas/core/kadascoordinateformat.cpp

```diff
@@ -98,24 +98,27 @@ QString KadasCoordinateFormat::getDisplayString( const QgsPointXY &p, const QgsC
     }
     case Format::DegMinSec:
     {
-      if ( translateDirectionSuffixes )
-        return QgsCoordinateFormatter::format( pTrans, QgsCoordinateFormatter::FormatDegreesMinutesSeconds, 1 );
-      else
-        return QgsCoordinateFormatter::format( pTrans, QgsCoordinateFormatter::FormatDegreesMinutesSeconds, 1, QgsCoordinateFormatter::FlagDegreesUseUntranslatedStringSuffix );
+      QgsCoordinateFormatter::FormatFlags formatFlags = QgsCoordinateFormatter::FormatFlag::FlagDegreesUseStringSuffix;
+      if ( !translateDirectionSuffixes )
+        formatFlags |= formatFlags | QgsCoordinateFormatter::FormatFlag::FlagDegreesUseUntranslatedStringSuffix;
```

Comment:
```suggestion
        formatFlags |= QgsCoordinateFormatter::FormatFlag::FlagDegreesUseUntranslatedStringSuffix;
```

IIUC this is the same?